### ISSUE TITLE
URL not showing in Publications Section

### ISSUE THREAD

**Issue by schrummy14** [sinaatalay/RenderCV#128]
**Describe the bug**
When using a URL instead of the DOI, no URL is shown in the PDF.

Looking at the .md file that is created, the url is used to produce a name for the url but no url is inserted
```markdown
## My Awesome Thing That Uses a URL ([https://www.example.com/]())
- 2020
- **schrummy14**
```

rendercv version 1.12

**To Reproduce**
```yaml
    publications:
      - title: My Awesome Thing That Uses a URL
        authors:
          - '**schrummy14**'
        date: 2020
        url: https://www.example.com/
```

**Screenshots**
Using URL:
![image](https://github.com/user-attachments/assets/ffba65d4-34a4-4a6b-ad61-a041b8acbc50)

Using DOI:
![image](https://github.com/user-attachments/assets/368c0be7-274f-42c3-83d4-0dd09e01fa8f)

**Comment by sinaatalay** [comment]
Thank you! The fix will be released in the next release.

**Comment by schrummy14** [comment]
Hello,
Thanks for the response. I was able to get the url to show using the below settings as a work around.
```yaml
    publications:
      - title: My Awesome Thing That Uses a URL
        authors:
          - '**schrummy14**'
        date: 2020
        url: https://www.example.com/
        clean_url: https://www.example.com/
```

**Comment by sinaatalay** [comment]
Yes, this can be used as a temporary fix.

**Comment by sinaatalay** [comment]
This issue has been fixed in `v1.13`.

**Comment by schrummy14** [comment]
Thank you
